src/mscorlib/src/System/Decimal.cs

```diff
@@ -329,13 +327,13 @@ public static Decimal FromOACurrency(long cy)
             }
             throw new ArgumentException(Environment.GetResourceString("Arg_DecBitCtor"));
         }
-    
-        // Returns the absolute value of the given Decimal. If d is
-        // positive, the result is d. If d is negative, the result
-        // is -d.
+
+        // Returns the absolute value of the given decimal. If value is
+        // positive, the result is value. If value is negative, the result
+        // is -value.
         //
-        internal static Decimal Abs(Decimal d) {
-            return new Decimal(d.lo, d.mid, d.hi, d.flags & ~SignMask);
+        public static decimal Abs(decimal value) {
+            return new decimal(value.lo, value.mid, value.hi, value.flags & ~SignMask);
```

Comment:
As much as I would love to fix some of these parameter naming issues, we tried such changes in the past and they just broke too much code. People calling these using call-by-name convention.